CMakeLists.txt

```diff
@@ -32,7 +32,7 @@ set(LIBRARY_OUTPUT_PATH ${PROJECT_SOURCE_DIR}/lib)
 #SET(CMAKE_CXX_FLAGS "-ggdb -O3 -fPIC -std=c++0x")
 #SET(CMAKE_CXX_FLAGS "-ggdb -O3 -fPIC")
 SET(CMAKE_BUILD_TYPE RELEASE)
-#add_compile_options(-std=c++11)
+add_compile_options(-std=c++11)
```

Comment:
I turned this off deliberately. If I remember correctly, because of issues with the pcl library and its boost dependencies, which crashed in the static initialization phase when the headers aren't compiled with the same c++ version as the libraries.
See #8 

Maybe this occurs only on older systems. On which system did you test your changes?